### PR TITLE
Pricing updates

### PR DESCRIPTION
**Pull request by nevercode-kevin**
- Add steps that explain how to subscribe to the Professional plan from within the web app
- Add details that the Professional Plus plan can also be paid for by invoice.

### CODE REVIEW

File: content/billing/pricing.md [hunk 1]
```diff
@@ -95,13 +95,24 @@ The Codemagic Professional plan gives you a fixed price plan with the following
...
 
 An **annual** subscription with **20% discount** is **$2,870/year**. Additional concurrencies are **$960/year**. 
 
-The Professional plan can only be paid for with credit card. If you require annual invoicing please refer to the Enterprise plan below.
+The Professional plan can only be paid for with credit card.
 
-If you would like to upgrade to a Professional plan, please contact us [here](https://codemagic.io/contact/).
+If you have already created a Team in the Codemagic web app, you can upgrade to the Professional plan as follows:
+
+1. Log into Codemagic.
```
Comment (helinanever) [inline content/billing/pricing.md:106]: Is it "into" or "in to"?

File: content/billing/pricing.md [hunk 2]
```diff
@@ -95,13 +95,24 @@ The Codemagic Professional plan gives you a fixed price plan with the following
...
+2. Click 'Billing' in the left-hand menu.
+3. Click on your Team account.
+4. Click on the 'Enable Billing' button.
```
Comment (helinanever) [inline content/billing/pricing.md:109]:
```suggestion
4. Click on the **Enable billing** button.
```

File: content/billing/pricing.md [hunk 3]
```diff
@@ -95,13 +95,24 @@ The Codemagic Professional plan gives you a fixed price plan with the following
...
+5. Enter your **credit card** details and **company information**.
+6. Once you have entered your details the pay-as-you-go billing will be enabled. To upgrade to the annual Professional plan click on the ‘Upgrade’ button and follow the instructions to subscribe to the Professional plan.
+
+If you require additional concurrencies or have any other questions about the Professional plan, please contact us [here](https://codemagic.io/contact/).
+
+If you require annual invoicing please refer to the Professional Plus and Enterprise plans below.
```
Comment (helinanever) [inline content/billing/pricing.md:115]:
```suggestion
If you require annual invoicing, please refer to the Professional Plus and Enterprise plans below.
```

**Review by helinanever — CHANGES_REQUESTED** [review]

File: content/billing/pricing.md (hunk 1 above)
Comment (nevercode-kevin) [reply content/billing/pricing.md:106]: Lets use "in to"

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "statusBarItem.hoverBackground": "#429151",
+        "statusBarItem.remoteBackground": "#54b266",
+        "statusBarItem.remoteForeground": "#15202b",
+        "titleBar.activeBackground": "#54b266",
+        "titleBar.activeForeground": "#15202b",
+        "titleBar.inactiveBackground": "#54b26699",
+        "titleBar.inactiveForeground": "#15202b99"
+    },
+    "peacock.color": "#54b266"
+}
```
[the remark this answers is not in the document]
Comment (nevercode-kevin) [reply .vscode/settings.json]: Ah, no!

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**